src/plugins/workspace/public/components/workspace_form/workspace_detail_form.tsx

```diff
@@ -34,7 +33,7 @@ interface FormGroupProps {
 const FormGroup = ({ title, children, describe }: FormGroupProps) => (
   <>
     <EuiFlexGroup gutterSize="xl">
-      <EuiFlexItem grow={false} className="workspace-detail-form-group">
+      <EuiFlexItem grow={false} style={{ width: '15%' }}>
```

Comment:
Why do we need this?

Reply:
The width is to maintain the spacing between title and child ：
<img width="1449" alt="image" src="https://github.com/user-attachments/assets/3057d369-0a55-4935-9023-b298348afee6">
If there is no width：
![image](https://github.com/user-attachments/assets/a68faae0-c70e-4015-a3a6-01151138d746)